stream.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Oscillator stream
+ * http://webaudio.github.io/web-audio-api/#the-oscillatornode-interface
+ *
+ * @module  audio-oscillator
+ */
+'use strict';
+
+var direct = require('./direct')
+var Readable = require('readable-stream')
```

Comment:
Okay, fair.  I did try a version with `audio-through` as well but i got same recursion problems.

I used `Readable` because was confused about if you could use `audio-through` that way after the errors.